NpingOps.cc won't compile without limits.h header as INT_MAX is not defined.

This was offered upstream:
https://github.com/nmap/nmap/pull/2509

--- nmap-7.92/nping/nping.h
+++ nmap-7.92/nping/nping.h
@@ -71,6 +71,7 @@
 #include <stdarg.h>
 #include <errno.h>
 #include <ctype.h>
+#include <limits.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 
